01/implement-most-frequent-k-elements.instructor.md

```diff
@@ -0,0 +1,29 @@
+# Instructor: Implement Most Frequent K Elements
+
+An example of a working implementation:
+
+```python
+def top_k_frequent_elements(arr, k):
+    if len(arr) == 1: return [arr[0]]
+    
+    hash_table = {}
+    result = []
+
+    for num in arr:
+        if hash_table.get(num, False):
+            hash_table[num] += 1
+            
+        else:
+            hash_table[num] = 1
+            result.append(num)
+            
+    i = 0
+    while i < len(result) - 1:
+        for j in range(len(result) - 1):
```

Comment:
If we keep this written out, this looks like bubble sort to me? In which case, I think the inner loop only  needs to run up to `len(result) - i - 1`.

Reply:
✅